packages/data-provider/src/controllers/dictionaryController.ts

```diff
@@ -13,18 +13,20 @@ const controller = (dependencies: BaseDependencies) => {
 			try {
 				const categoryName = req.body.categoryName;
 				const dictionaryName = req.body.dictionaryName;
-				const dictionaryVersion = req.body.version;
+				const version = req.body.version;
```

Comment:
would this be a dictionary version? or some other item's version? think the variable name could be a bit more descriptive of what it holds

Reply:
This comment will be handled in PR[#105](https://github.com/overture-stack/lyric/pull/105)
Changed variable name to more descriptive name `dictionaryVersion`